plugins/query.rst

```diff
@@ -0,0 +1,26 @@
+Query plugin
+==============
```

Comment:
i think this is now off too, needs to be same lenght as title

Reply:
Fixed